overlord/snapstate/snapmgr.go

```diff
@@ -124,8 +127,24 @@ func (m *SnapManager) doPrepareSnap(t *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
+	if ss.Revision == 0 { // sideloading
+		// to not clash with not sideload installs
+		// and to not have clashes between them
+		// use incremental revisions starting at 100000
+		// for sideloads
+		revision := snapst.LastSideloadRevision
+		if revision == 0 {
+			revision = firstSideloadRevision
```

Comment:
Can we please +1 the base, so we start those on *1 too, same as the store revisions.